### DISCUSSION TITLE
Is it posible to configure the language / text direction for a single Markdown file?

### DISCUSSION THREAD

**Discussion by Lucas-C** [squidfunk/mkdocs-material#4465] [Q&A] [answered]
Hi!

Thank you for providing & maintaing this excellent theme!

In the `fpdf2` open source project, we have a tutorial translated in various language.
A contributor recently provided an Hebrew translation, using `<div dir="rtl">` to set the proper text direction.
However this breaks the Markdown parsing: https://pyfpdf.github.io/fpdf2/Tutorial-he.html

Hence my question: is it posible to configure the language / text direction for a single Markdown file?
Hopefully through YAML metada or `mkdocs.yml`?

I have read the documentation on [Changing the language](https://squidfunk.github.io/mkdocs-material/setup/changing-the-language/),
the discussion on [Multilingual setup](https://github.com/squidfunk/mkdocs-material/discussions/2346) and search the _issues_ & _discussions_ about this,
but could not find any answer so far.

**Comment by Lucas-C** [comment, marked as answer]
For reference, using the [markdown-in-html](https://squidfunk.github.io/mkdocs-material/setup/extensions/python-markdown/#markdown-in-html) Markdown extension did not solve the issue

**Reply by facelessuser** [reply]
> For reference, using the [markdown-in-html](https://squidfunk.github.io/mkdocs-material/setup/extensions/python-markdown/#markdown-in-html) Markdown extension did not solve the issue

When you say it didn't help, did you utilize it properly?

```html
<div dir="rtl" markdown="1">
```

**Reply by facelessuser** [reply]
Seems to work for me when I enable markdown-in-html and use `<div dir="rtl" markdown="1">`. I just copied the content of that page into a local project to see how it was rendered.

<img width="748" alt="Screen Shot 2022-10-06 at 12 05 50 PM" src="https://user-images.githubusercontent.com/1055125/194386850-04824914-019d-4239-bc0d-79da848caad4.png">

**Reply by Lucas-C** [reply]
Wonderful, thank you!
Problem solved 🥳

I did not realize that `markdown="1"` was also required...
Is this documented somewhere?

Thank you for your help @facelessuser,
and for the time you took to test this!

**Reply by facelessuser** [reply]
Yes, Material actually links to the specified extensions documentation: https://python-markdown.github.io/extensions/md_in_html/. You should find everything you need to know there. Material mainly mentions what it supports and how to enable it, but doesn't go into exhaustive documentation on how to use the extension, usually, you have to read that extension's official documentation for more info.
